### ISSUE TITLE
Dependencies warning after updating akita to version 3.5.5

### ISSUE THREAD

**Issue by johanrin** [NetanelBasal/akita-ng-forms-manager#5]
I have the following warning after updating akita to version 3.5.5

```
$ npm install @datorama/akita
npm WARN @datorama/akita-ng-forms-manager@3.0.0 requires a peer of @datorama/akita@~3.0.0-alpha.3 but none is installed. You must install peer dependencies yourself.

+ @datorama/akita@3.5.5
```

Is **akita-ng-forms-manager** package compatible with the latest version of akita?

**Comment by NetanelBasal** [comment]
Yes, I will fix it.

**Comment by NetanelBasal** [comment]
Fixed in 3.0.3
